app/redux/store.js

```diff
@@ -31,28 +31,31 @@ const zmqMiddlewareCreator = (zmq, subzmq) => {
 
       switch (type) {
         case 'CONNECT':
-          zmq.connect(`tcp://${sock.ip || 'localhost'}:5556`);
-          subzmq.connect(`tcp://${sock.ip || 'localhost'}:5555`);
+          reqzmq.connect('tcp://localhost:5556');
+          subzmq.connect('tcp://localhost:5555');
 
           break;
         case 'COMMAND':
-          if (sock.cmd !== "handshake") {
+          if (zmq.cmd && zmq.cmd.indexOf('start') === 0) {
             subzmq.subscribe('');
           }
-          zmq.send(sock.cmd);
+          if (zmq.cmd && zmq.cmd.indexOf('stop') === 0) {
+            subzmq.unsubscribe('');
+          }
+
+          reqzmq.send(zmq.cmd);
           break;
         case 'DISCONNECT':
-          subzmq.unsubscribe();
+          subzmq.unsubscribe('');
 
-          zmq.disconnect();
-          subzmq.disconnect();
+          reqzmq.disconnect('tcp://localhost:5556');
+          subzmq.disconnect('tcp://localhost:5555');
           break;
```

Comment:
Could be moved to a constant, or some config file? Hardcoding directly where its used seems weird